### ISSUE TITLE
Feature: Interface for creating custom light settings

### ISSUE THREAD

**Issue by Muttley** [Muttley/foundryvtt-shadowdark#383]
These could then be assigned to Light Source items and Effects.

**Comment by jkwatson** [comment]
Yes! This would be great to have. I don't really care for the defaults, and would love to customize things so the default light settings (color, dropoff profile, animations, etc) could be different.
